rock/python.rb

```diff
@@ -216,4 +228,26 @@ def self.activate_python_path(pkg,
                              "python#{version}","site-packages")
         [bin, version, path]
     end
+
+    def self.setup_python_configuration_options(ws: Autoproj.workspace)
+        ws.config.declare 'USE_PYTHON', 'boolean',
+            :default => 'no',
+            :doc => [ "Do you want to activate python?" ]
+
+        value = ws.config.get('USE_PYTHON')
+        if value && !Autoproj::BuildOption::FALSE_STRINGS.include?(value)
```

Comment:
'boolean' should take care of the conversion. Doesn't it ?

Reply:
It does not for autoproj <= v.2.11.0 when using noninteractive mode. Fortunately the bug has already been fixed for upcoming releases https://github.com/rock-core/autoproj/commit/efcd09dcd918281a3041b39a8f6defa3e2d7f2ea

Reply:
K. I need to release that stuff.